### ISSUE TITLE
Crash when log off from Windows

### ISSUE THREAD

**Issue by denisglu** [ShareX/ShareX#3780]
Error dialog with crash info appears when I log off from Windows.
No problem when exit application via menu.
The problem started to occur after recent auto-update to 12.3. 
Stable.
Windows 7 x64.

Crash log:
```
2018-11-22 10:20:05.530 - WatchFolderManager started.
2018-11-22 11:06:20.248 - Unhandled exception:
System.OverflowException: Arithmetic operation resulted in an overflow.
   at ShareX.MainForm.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

ShareX 12.3 (Release)

```

**Comment by sylveon** [comment]
Fixed in 12.3.1

**Comment by Jaex** [comment]
Duplicate of #3738
